_episodes/16-parallel.md

```diff
@@ -32,56 +397,34 @@ this is a common tool on HPC systems.
 {: .callout}
 
 MPI jobs cannot generally be run as stand-alone executables. Instead, they should be
```

Comment:
We actually ran into this a few times and in fact there's nothing in the standard that requires that a a parallel program can run outside of an MPI environment. We've had this issue with both Intel MPI and ParaStation MPI at one point, usually they still fix this though because it is clearly useful.